stories/virtru/sendMessage/basic.jsx

```diff
@@ -0,0 +1,168 @@
+              <div style={{ padding: '0 12px 12px' }}>
+                <Space direction="vertical" style={{ width: '100%' }}>
+                  <Row justify="space-between">
+                    <Col>{fileList.length} File(s) Added</Col>
+                    <Col>
+                      <Button size="small" disabled={fileList.length === 0}>
+                        Remove all
+                      </Button>
+                    </Col>
+                  </Row>
+                  {fileList.length > 0 && (
+                    <Row gutter={[12, 12]}>
+                      {fileList.map((file) => {
+                        return (
+                          <Col key={file.name} md={12}>
+                            <FileListItem
+                              name={file.name.split('.')[0]}
+                              extension={`.${file.name.split('.')[1]}`}
+                              size="1.09 Kb"
+                              remove={handleClickRemove}
+                              tooltipText="This file is encrypted with no additional security options enabled."
+                              warning={false}
+                            />
+                          </Col>
+                        );
+                      })}
+                    </Row>
+                  )}
+                </Space>
+              </div>
+            </Space>
+          </GroupEditor>
+        </Space>
+      }
+      right={
+        <div
+          style={{
+            display: 'flex',
+            height: '100%',
+            flexDirection: 'column',
+            justifyContent: 'space-between',
+          }}
+        >
+          <div>
+            <Title level={5}>
+              Security <InfoTooltip title="security" />
+            </Title>
+            <div>MESSAGE OPTIONS</div>
+            <ItemOption
+              icon={<NoForwardingIcon />}
+              name="Disable Forwarding"
+              changeSwitch={changeSwitch}
+              tooltipTitle="Some tooltip text"
+            />
+            <ItemOption
+              icon={<HourglassIcon />}
+              name="Expiration Date"
+              changeSwitch={changeSwitchExpirationPicker}
+              tooltipTitle={<span>text in html tag</span>}
+            />
+            {showExpirationPicker && <ExpirationPicker getTime={getTime} />}
+            <Divider plain />
+            <div>
+              ATTACHMENT OPTIONS <InfoTooltip title="Some tooltip" />
+            </div>
+            <ItemOption icon={<WatermarkIcon />} name="Watermarking" changeSwitch={changeSwitch} />
+            <ItemOption
+              icon={<WatermarkIcon />}
+              name="Persistent protection"
+              changeSwitch={changeSwitch}
+            />
+          </div>
+          <Row justify="end">
+            <Space>
+              <Button>Cancel</Button>
+              <Button type="primary">Send</Button>
+            </Space>
+          </Row>
+        </div>
+      }
+    />
```

Comment:
I think the idea behind CCR-850 is to create a component called something like Composer that would take props from Secure Share and Secure Reader.

So in this file, you would only import Composer component and any components you wanted to use as props for Composer.